docs/MAC_SETUP.md

```diff
@@ -0,0 +1,103 @@
+# Mac Development Setup
+
+## Application Installs
+
+* Install [Homebrew](https://brew.sh/) this will prompt for a bunch of passwords entries. It is working just get to the end!
+* Install [Docker Desktop](https://docs.docker.com/desktop/install/mac-install/)
+* Install [VSCode](https://formulae.brew.sh/cask/visual-studio-code)
+    * Install [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension
+
+## GitHub Setup
+
+Create an ssh keypair:
+
+```
+ssh-keygen -t rsa -a 100 -Z aes128-gcm@openssh.com
+```
+
+Create a file `config` in the ~/.ssh folder:
+
+```
+touch ~/.ssh/config
+```
+
+Add the following to the file:
+
+```
+Host github.com
+    AddKeysToAgent yes
+    User git
+    PubkeyAcceptedAlgorithms +ssh-rsa
+```
+
+## Standup Dev Playground
+
+### TL;DR
+
+This repo includes a script that does the setup steps below checking that your environment is setup correctly 
+and if so running the commands to stand up the Docker network and dev playground.
+
+```shell
+$> bin/devup
+```
+
+### Create docker network
+
+```
+docker network create --driver bridge --ipv6  --subnet fd00::/80 infranet_network --attachable
+```
+
+### Start up the docker EVerest Playground
+
+docker compose -f ./everest-utils/docker/docker-compose.yml" up -d mqtt-server 
+docker compose -f ./everest-utils/docker/docker-compose.yml" up -d nodered
+
+## Start VSCode Docker Dev Environment
+
+In VSCode:
+
+* Press `CMD + Shift + P`
+* type `Dev Containers: Open Folder in Container...`
+* Navigate and open the docker/everest-playground folder. 
+
+This will open the EVerest Playground as a VSCode dev container. You should now be ready to go.
```

Comment:
I have never used dev containers before - I edit my code using vim 😄 - but this does seem to be very useful for those who use vscode. But doesn't this need a `devcontainer.json` to work? I don't see the file in the PR.

Or do we not need it because we don't need any additional dependencies/plugins?

Reply:
The [`devcontainer.json`](https://github.com/US-JOET/everest-utils/blob/APIT-130_improve_workflow/docker/everest-playground/.devcontainer.json) file is in the same folder as the [`Dockerfile`](https://github.com/US-JOET/everest-utils/tree/APIT-130_improve_workflow/docker/everest-playground) that we're opening up.

Reply:
Respect 😄. When I grow up, I want to be a neovim developer.